src/flp.rs

```diff
@@ -752,6 +747,225 @@ pub(crate) fn gadget_poly_len(gadget_degree: usize, wire_poly_len: usize) -> usi
     gadget_degree * (wire_poly_len - 1) + 1
 }
 
+/// Utilities for testing FLPs.
+#[cfg(feature = "test-util")]
+pub mod test_utils {
```

Comment:
Would you use this outside of `prio`? Which is to say, does `pub(crate)` suffice? Same goes for the various `pub` items defined in this module.

Reply:
I intend to use this outside of `prio` (see linked issue)

Reply:
Good enough for me, thank you for confirming.